### ISSUE TITLE
Acc Tests: OpenLab Failure

### ISSUE THREAD

**Issue by jtopjian** [gophercloud/gophercloud#2168]
The OpenLab tests have recently begun failing. I think this is due to the tests using Ubuntu Bionic, which may not be supported in Devstack any more.

**Comment by nikParasyr** [comment]
@jtopjian just for reference:
If I understand openlab correctly, devstack from master branch is used to build the test env. From checking the history on master branch:

As you say Bionic support was dropped recently on devstack: https://opendev.org/openstack/devstack/commit/7ad4cd07c8bf4f302acc4fc6684e362309332c9d

Another change that I noticed that might be important/worth noting: https://opendev.org/openstack/devstack/commit/35cec0d7c0857d76d3ea0b52b97f2a166c04c13e. I'm not sure how to handle the tests for blockstorage/v2

**Comment by zaneb** [comment]
```
+ ./stack.sh:main:230                      :   SUPPORTED_DISTROS='focal|f31|f32|opensuse-15.2|opensuse-tumbleweed|rhel8'
+ ./stack.sh:main:232                      :   [[ ! bionic =~ focal|f31|f32|opensuse-15.2|opensuse-tumbleweed|rhel8 ]]
+ ./stack.sh:main:233                      :   echo 'WARNING: this script has not been tested on bionic'
WARNING: this script has not been tested on bionic
+ ./stack.sh:main:234                      :   [[ '' != \y\e\s ]]
+ ./stack.sh:main:235                      :   die 235 'If you wish to run this script anyway run with FORCE=yes'
```

So ideally OpenLab should switch to focal. Short-term workarounds might be to pass `FORCE=yes` in the environment or to use the `stable/wallaby` branch of devstack.

**Comment by jtopjian** [comment]
> Another change that I noticed that might be important/worth noting

Good catch. The current error I'm hung up on isn't related to Block Storage v2, but I imagine that will need addressed before #2169 is merged. Let's wait until the iSCSI issue is sorted and see if there's a specific error about v2 support.

> So ideally OpenLab should switch to focal.

Right. This is controllable by the `zuul.yml` file within Gophercloud and in queue here: https://github.com/gophercloud/gophercloud/pull/2169

The next issue seems to be some kind of iSCSI error. I'm waiting for the OpenLab team to take a look at the issue I've opened here (https://github.com/theopenlab/openlab/issues/751) before digging any further.

It's possible to pass environment variables, so we _could_ do `FORCE=yes` and see what happens, but I want to give the OpenLab team a few days to look at the iSCSI issue being reported.

**Comment by jtopjian** [comment]
Fixed in #2169
